images/win/scripts/Installers/Install-Haskell.ps1

```diff
@@ -0,0 +1,29 @@
+################################################################################
+##  File:  Install-Haskell.ps1
+##  Desc:  Install Haskell for Windows
+################################################################################
+
+# Get last 3 latest versions of GHC
+$ChocoVersionsOutput = & choco search ghc --allversions --limit-output
+$VersionsList = $ChocoVersionsOutput | Where-Object { $_.StartsWith("ghc|") } | ForEach-Object { $_.TrimStart("ghc|") } `
+| ForEach-Object { [Version]::Parse($_) } | Sort-Object -Descending
+
+$MajorMinorVersionsToInstall = $VersionsList | ForEach-Object { [System.Version]::Parse($_.ToString(2)) } `
+| Get-Unique | Sort-Object -Descending | Select-Object -First 3
+
+$LatestGhcVersions = $MajorMinorVersionsToInstall | ForEach-Object {
+    $version = $_
+    return $VersionsList | Where-Object {
+        return ($_.Major -eq $version.Major) -and ($_.Minor -eq $version.Minor)
+    } | Select-Object -First 1
+}
```

Comment:
Can we simplify this logic a bit via `Group-Object`?
```
$MajorMinorVersionsToInstall  = $VersionsList | Group-Object { $_.ToString(2) } | 
...
```

So you will have groups where key will version with 2 numbers (like `3.8`) and value is list of available versions (like `3.8.1, 3.8.2,3.8.3`).
And then you will pickup latest version from each group and pickup 3 latest groups